targets/targets.json

```diff
@@ -4258,6 +4302,10 @@
             "lse_drive_load_level": {
                 "help": "HSE drive load level  RCC_LSEDRIVE_LOW | RCC_LSEDRIVE_MEDIUMLOW | RCC_LSEDRIVE_MEDIUMHIGH | RCC_LSEDRIVE_HIGH",
                 "value": "RCC_LSEDRIVE_LOW"
+            },
+            "i2c_timing_value_algo": {
+                "help": "If value was set I2C timing algorithm is enabled. Enabling may leads to performance issue. Keeping this false and changing system clock will cause compilation error)",
+                "value": true
```

Comment:
Why this value is true ?

Reply:
No missed after testing. changed to false